src/cs/production/Katabasis.Framework/Katabasis/Audio/SoundEffectInstance.cs

```diff
@@ -242,30 +243,30 @@ public virtual void Play()
 			/* Create handle */
 			if (_isDynamic)
 			{
-				_FAudio.FAudioSourceVoice* handle;
-				_FAudio.FAudio_CreateSourceVoice(
-					(_FAudio.FAudio*)dev.Handle,
+				FAudio.FAudioSourceVoice* handle;
+				FAudio.FAudio_CreateSourceVoice(
+					(FAudio.FAudioSystem*)dev.Handle,
```

Comment:
Glad to see you found out what I did with `FAudio` type in C being alias to `FAudioSystem` in C#. This is exactly what I expected to change for Katabasis.